sky/execution.py

```diff
@@ -632,6 +632,19 @@ def spot_launch(
     dag_utils.fill_default_spot_config_in_dag(dag)
 
     for task_ in dag.tasks:
+
```

Comment:
double check if we need to overwrite the params here.

Reply:
edited